tools/test_up.sh

```diff
@@ -212,8 +237,8 @@ echo cp -r "${testdir}/${port}" "${testdir}/previous"
 cp -r "${testdir}/${port}" "${testdir}/previous"
 
 new_args+=( --verify-in "${testdir}/verify_file" )
-echo "$ct" repair -g "$gen" -q "${new_args[@]}"
-if ! "$ct" repair -g "$gen" -q "${new_args[@]}"; then
+echo "$ct" fill -g "$gen" -q "${new_args[@]}"
+if ! "$ct" fill -g "$gen" -q "${new_args[@]}"; then
```

Comment:
Is this change intentional?

Reply:
Yes, this was one of the test bugs.
Sometimes the repair test would not actually change the downstairs that we dump below.
The `fill` touches every extent, so we know there should be a difference when we dump.